crates/gitbutler-branch-actions/src/move_commits.rs

```diff
@@ -163,12 +183,12 @@ fn check_source_lock(
 }
 
 /// determines if the source commit is locked to any of its ancestors
-fn check_source_lock_to_ancestors(
+fn check_source_lock_to_commits(
     repository: &git2::Repository,
-    ancestor_commits: Vec<git2::Commit>,
+    ancestor_commits: &Vec<git2::Commit>,
     source_commit_diff: &HashMap<std::path::PathBuf, Vec<gitbutler_diff::GitHunk>>,
```

Comment:
Nice!
Now let's get rid of all references to the word "ancestor", please.
Since this function is now 'generic' we can refer to `commits` instead of `ancestor_commits`.